include/dslash_helper.cuh

```diff
@@ -264,13 +266,31 @@ namespace quda
 
     int pack_threads; // really number of face sites we have to pack
     int_fastdiv blocks_per_dir;
+    int sites_per_block;
     int dim_map[4];
     int active_dims;
     int pack_blocks; // total number of blocks used for packing in the dslash
+    int ext_dims;    // dimension to run in the exterior Dslash
+    int ext_blocks;
```

Comment:
Can this be renamed to `exterior_blocks`, etc for other `ext_*` variables for code readability?

Reply:
done